validation/data-availability-type_test.go

```diff
@@ -0,0 +1,45 @@
+package validation
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func testDataAvailabilityType(t *testing.T, chain *ChainConfig) {
+	rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+	require.NotEmpty(t, rpcEndpoint)
+
+	client, err := ethclient.Dial(rpcEndpoint)
+	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+	// Standard chains must be eth-da for now. Will add checks for alt-da later
+	require.Equal(t, superchain.EthDA, chain.DataAvailabilityType)
+
+	batchSubmitterAddress := Addresses[chain.ChainID].BatchSubmitter
+	require.NotZero(t, batchSubmitterAddress)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	depth := chain.SequencerWindowSize
+	require.NotZero(t, depth)
+
+	// First attempt without needing an archive node
+	//  - full nodes keep the last 128 blocks but we use 120 to give a buffer
+	nonArchivalDepth := 120
+	_, found, err := eth.CheckRecentTxs(ctx, client, nonArchivalDepth, common.Address(batchSubmitterAddress))
+	if !found {
+		_, found, err = eth.CheckRecentTxs(ctx, client, int(depth), common.Address(batchSubmitterAddress))
+	}
+	require.NoErrorf(t, err, "failed when checking chain for recent batcher txs from %s", batchSubmitterAddress)
```

Comment:
I think we should actually find that tx and check that it was sent to the batcher inbox. That's an additional cheap sanity check we can do here.

Reply:
https://github.com/ethereum-optimism/superchain-registry/pull/421/commits/ce5c6a61bbf97927aed11245602756a001bc76bb